rbac/management/cache.py

```diff
@@ -34,6 +34,96 @@ def connection(self):
                 raise
         return self._connection
 
+    @contextlib.contextmanager
+    def delete_handler(self, err_msg):
+        """Handle delete events."""
+        try:
+            yield
+        except exceptions.RedisError:
+            logger.exception(err_msg)
+
+    def get_from_redis(self, index):
+        """Get object from redis based on index."""
+        obj = self.connection.hget(*(self.key_for(index[0]), index[1]))
+        if obj:
+            return json.loads(obj)
+
+    def get_cached(self, index, error_message):
```

Comment:
The `index` naming in this call chain confused me for a bit, because it doesn't appear to be the actual index value, but rather the key?

Reply:
Agree, changing it to key